lib/pages/search/recent_contacts_banner_widget.dart

```diff
@@ -41,20 +43,19 @@ class PreSearchRecentContactsContainer extends StatelessWidget {
 }
 
 class PreSearchRecentContactWidget extends StatelessWidget {
-  final SearchController searchController;
-  final User user;
+  final Room room;
   const PreSearchRecentContactWidget({
     super.key,
-    required this.user,
-    required this.searchController,
+    required this.room,
   });
 
   @override
   Widget build(BuildContext context) {
+    final displayName = room.getLocalizedDisplayname(
+      MatrixLocals(L10n.of(context)!),
+    );
     return InkWell(
-      onTap: () {
-        searchController.goToChatScreenFormRecentChat(user);
-      },
+      onTap: () => context.go('/rooms/${room.id}'),
```

Comment:
still here https://github.com/linagora/twake-on-matrix/pull/1800#discussion_r1615447655

Reply:
no need to depend on the controller, its will make the component harder to reuse.

Reply:
alright then